for_cr.ipynb

```diff
@@ -24,7 +24,7 @@
     }
```

Comment:
<p>well 6/6 is 1 not 0.8333334</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/helkhalfi/bigbiggy/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>